app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = MerchantsFacade.find_all
+  end
+
+  def show
+    # merchants = MerchantsFacade.find_all
+    # @merchant = merchants.find(params[:id])
+    # require "pry"; binding.pry
+
+    @merchant = MerchantsFacade.find_one(params[:id])
+    @items = MerchantItemsFacade.find_all(params[:id])
+  end
```

Comment:
Nice job here. You could also consider having a MerchantItemsFacade, and you could create MerchantItem objects to send to the view -- this could limit how much you're sending to the view (from two variables to two)